applications/new_schema/types.py

```diff
@@ -123,6 +123,9 @@ class WinterStorageApplicationFilter(django_filters.FilterSet):
     no_customer = django_filters.BooleanFilter(
         field_name="customer", lookup_expr="isnull"
     )
+    area_type = django_filters.ChoiceFilter(
+        choices=[(entry, entry.value) for entry in ApplicationAreaType]
+    )
```

Comment:
i think a cleaner approach if we use graphene enum here, like we do for orders with [`OrderTypeEnum`](https://github.com/City-of-Helsinki/berth-reservations/blob/6da06ad55793248cb6d822f7afa689006ac6e7c3/payments/schema/queries.py#L64)

Reply:
this way we ensure that the client know what values are allowed for this filter and they are parsed correctly

Reply:
done